ydb/core/tx/replication/ut_helpers/test_topic.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <util/generic/maybe.h>
+#include <util/generic/ptr.h>
+#include <util/generic/string.h>
+#include <util/generic/vector.h>
```

Comment:
```suggestion
#include <util/generic/ptr.h>
#include <util/generic/string.h>
```

Остальное лишнее (полагаю, результат копи-пасты из test_table.h).